src/commands/misc/covid/countries.js

```diff
@@ -18,21 +18,23 @@ module.exports = class CovidCountries extends Command {
   async run ({ t, author, channel, language }, country) {
     moment.locale(language)
     try {
-      const { data } = await this.client.apis.covid.getCountry(country)
+      const countryData = await this.client.apis.covid.getCountry(country)
+      const countryVaccinated = await this.client.apis.covid.getCountryVaccinated(country)
```

Comment:
Should probably use Promise.all here. Don't think there's any reason we have to wait for the first API call to resolve before making the next one.